src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -158,6 +160,7 @@
     <Reference Include="System.Text.Encoding.Extensions" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Collections.Immutable\src\System.Collections.Immutable.csproj" />
```

Comment:
Has this been updated? I got the impression this had been removed...

Reply:
Yes, forgot to remove it here. Pushed a commit removing it.